src/app/views/communities/view/overview/overview.ts

```diff
@@ -165,6 +208,30 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 		);
 		this.knownMembers = overviewPayload.knownMembers || [];
 		this.knownMemberCount = overviewPayload.knownMemberCount || 0;
+
+		Meta.description = this.$gettextInterpolate(
+			// tslint:disable-next-line:max-line-length
+			'Welcome to the %{ name } community on Game Jolt! Find and explore %{ name } fan art, lets plays and catch up on the latest news and theories!',
```

Comment:
I don't think we should duplicate name twice. Also no need to say find and explore, since that's kind of hte same. I think: `Welcome to the %{ name } community on Game Jolt! Explore fan art, lets plays and catch up on the latest news and theories!`

Reply:
The purpose is to help with keyword density and to hint the search engines what this page is about. I think this should stay.